scipy/stats/_morestats.py

```diff
@@ -3904,3 +3904,71 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate', *,
     if not normalize:
         res *= (high-low)/(2.*pi)  # [1] (2.3.14) w/ (2.3.7)
     return res
+
+
+@_axis_nan_policy_factory(
+    lambda x: x, result_to_tuple=lambda x: (x,), n_outputs=1
+)
+def directionalmean(samples, *, axis=0):
+    """
+    Computes the directional mean of a sample of vectors.
+
+    Serves as equivalent of the sample mean for directional data whose
+    magnitude is irrelevant, e. g. unit vectors. The vectorial data
+    whose directionals means are computed must be located in the last
+    axis.
```

Comment:
Maybe add a sentence pointing users to `circmean` if they have angular data on the 2d plane.